docs/RULES.md

```diff
@@ -1,3 +1,48 @@
+
+<!-- vim-markdown-toc GFM -->
```

Comment:
I'd like to keep things editor agnostic. Would it be cool to remove this?

Reply:
Yes sure, go ahead. That is only meant to demarcate the toc area for a VIM plugin that does the heavy lifting of generating the toc. Please go ahead and remove it.